### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by ewanmay**
1. Links on References Tab to send you to URL properly
2. Geography tab aesthetic fix by removing unnecessary commas
3. Format date correctly on Details Modal
4. Implement "Database up to date to" properly by using it from back-end
5. Reconnect data on analyze bar charts.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -65,10 +65,15 @@ function App() {
     }
     
     const allFilterOptions = async () => {
-      const response = await api.getAllFilterOptions();
+      const { options, updatedAt, maxDate, minDate } = await api.getAllFilterOptions();
       dispatch({
```
Comment (atatmaja) [inline src/App.tsx:69]: would it be better to just consolidate this into a single dispatch?

**Review by atatmaja — APPROVED** [review]
LGTM just minor comments

**Review by brett-stephen — APPROVED** [review]
lookin good homie

File: src/App.tsx (hunk 1 above)
Comment (ewanmay) [reply src/App.tsx:69]: I like keeping the state updates distinct

**Pull request merged**